tuf/ngclient/_internal/requests_fetcher.py

```diff
@@ -53,7 +53,7 @@ def __init__(self):
         self.chunk_size: int = 400000  # bytes
         self.sleep_before_round: Optional[int] = None
 
-    def fetch(self, url, required_length):
+    def fetch(self, url: str, required_length: int) -> Iterator[bytes]:
```

Comment:
I guess this is more `max_length` than `required_length` as well?

the docstring could be better as well